### PR TITLE
OCR model with new training data

### PR DESCRIPTION
**Pull request by SaraWestWA**
# Description

Tesseract OCR model storysquad.traineddata is trained on data from --32 and about 50% of --52. The model accuracy increased from 12.93% to 48.69%; other parameters have not been fine-tuned. The data used for training, including .box and .lstmf files generated during training, can be found in storysquad-ground-truth.

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Change Status
- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested
- [ ] Yes
- [ ] No
- [x] Not necessary

## Checklist
- [x] There are no merge conflicts

### CODE REVIEW

**Comment by SaraWestWA** [comment]
Related Trello Card
https://trello.com/c/8IrgN008

**Review by nsriniva — DISMISSED** [review]
Please add some text to the README file to explain why the segmented images and associated text from the children's story submissions are OK to commit into github(and hence made public).

**Review by marianvinas — APPROVED** [review]

**Review by ahvblackwelltech — APPROVED** [review]
